modepy/tools.py

```diff
@@ -181,78 +193,81 @@ def inverse(self):
 EQUILATERAL_TO_UNIT_MAP: dict[int, AffineMap] = {
         1: AffineMap([[1]], [0]),
         2: AffineMap([
-            [1, -1/sqrt(3)],
-            [0,  2/sqrt(3)]],
-            [-1/3,   -1/3]),
+            [1, -1/math.sqrt(3)],
+            [0,  2/math.sqrt(3)]],
+            [-1/3,        -1/3]),
         3: AffineMap([
-            [1, -1/sqrt(3), -1/sqrt(6)],
-            [0,  2/sqrt(3), -1/sqrt(6)],
-            [0,         0,  sqrt(6)/2]],
-            [-1/2,   -1/2,       -1/2])
+            [1, -1/math.sqrt(3), -1/math.sqrt(6)],
+            [0,  2/math.sqrt(3), -1/math.sqrt(6)],
+            [0,              0,  math.sqrt(6)/2]],
+            [-1/2,        -1/2,            -1/2])
         }
 
 
-def equilateral_to_unit(equi):
+def equilateral_to_unit(equi: ArrayF) -> ArrayF:
     return EQUILATERAL_TO_UNIT_MAP[len(equi)](equi)
 
 
-def unit_vertices(dim):
-    warn("unit_vertices is deprecated. "
-            "Use modepy.unit_vertices_for_shape instead. "
-            "unit_vertices will go away in 2022.",
-            DeprecationWarning, stacklevel=2)
+def unit_vertices(dim: int) -> ArrayF:
+    warn("'unit_vertices' is deprecated. Use 'mp.unit_vertices_for_shape' instead. "
+         "'unit_vertices' will go away in 2022.",
+         DeprecationWarning, stacklevel=2)
```

Comment:
There's this and the `submesh` functions that have been deprecated for a long time. Remove?

Reply:
Yeah, kill them.

Another good (new-ish) option is to slap a `@typing_extensions.deprecated` decorator on there. That will start failing downstream bpr CIs (without impacting any code functionality).